api/src/services/graphqlInspector/graphqlInspector.ts

```diff
@@ -1,12 +1,42 @@
-import { coverage } from '@graphql-inspector/core'
+import { validate } from '@graphql-inspector/core'
```

Comment:
I'm not sure I'll keep all this functionality.

Reply:
Decided to remove functionally but got service to run converse without errors.

There are comments about next steps -- properly loading the source documents to apply coverage,

I tried a few methods (graphql-tools code file loader, glob filenames, etc) but I'll check with Kamal from the Guild actually how to do it. It expects a GraphQL "Source" which I thought was the last, but didn't quite work.